test/cbmc/stubs/sigv4_stubs.c

```diff
@@ -229,3 +229,30 @@ SigV4Status_t generateCanonicalAndSignedHeaders( const char * pHeaders,
 
     return returnStatus;
 }
+
+SigV4Status_t copyHeaderStringToCanonicalBuffer( const char * pData,
```

Comment:
It's fine to stub this but we need to start proving out functions that we stub separately in order to say that we truly have 100% coverage